rpc/events.go

```diff
@@ -9,11 +9,20 @@ import (
 	"github.com/NethermindEth/juno/jsonrpc"
 )
 
+const (
+	MaxBlocksBack        = 1024
```

Comment:
There is new constant introduced in Aneeque's PR `maxBlocksBack` in `rpc/handlers.go`. If both of them related consider to reuse existing one